### PR TITLE
Feature/responsive images

### PR DESCRIPTION
**Pull request by krystiangw**
For all images with ```data-fp-src``` attribute script construct source url with convert resize params.
This way image dimensions will be adjusted to image tag / window dimensions.
Additionally it can resize image on window resize.

### CODE REVIEW

File: src/utils/window.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        getWidth:getWidth,
+        getHeight:getHeight
+    };
+
+    /**
+        * Get the window width.
+        * @returns {number}
+    */
+    function getWidth() {
+        return document.documentElement.clientWidth || document.body && document.body.clientWidth || 1024;
```
Comment (exaroth) [inline src/utils/window.js]: i would suggest putting ` document.body && document.body.clientWidth` in brackets, for the code to be more readable.
``` js
return document.documentElement.clientWidth || (document.body && document.body.clientWidth) || 1024;
```

**Comment by exaroth** [comment]
:+1:

File: src/utils/window.js (hunk 1 above)
Comment (krystiangw) [inline src/utils/window.js]: Fixed.

**Comment by gr00by87** [comment]
:+1:

**Comment by gr00by87** [comment]
:+1:

**Pull request merged**